src/main/java/net/pms/dlna/virtual/MediaLibraryFolder.java

```diff
@@ -36,30 +56,38 @@ public void discoverChildren() {
 				if (expectedOutput == FILES) {
 					ArrayList<File> list = database.getFiles(sql);
 					if (list != null) {
-						UMSUtils.sort(list, PMS.getConfiguration().mediaLibrarySort());
+						UMSUtils.sort(list, PMS.getConfiguration().getSortMethod(null));
```

Comment:
Thanks for the review. We have a few custom sorts in here, along with dynamic string transformations before sorting so we can ignore certain character and words. I'm not familiar with doing that kind of thing with SQL but if you are interested in it please feel free to help